auxprogs/compileSpliceCands/Makefile

```
@@ -1,8 +1,9 @@
 compileSpliceCands : compileSpliceCands.o list.h list.o
```

```suggestion
CC?=gcc

compileSpliceCands : compileSpliceCands.o list.h list.o
```
add 
CC?=gcc
as in bam2wig Makefile

---

OK